### DISCUSSION TITLE
Windows ARM64 Support

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#858] [Ideas]
### What feature would you like to see?

I would like to see a native Windows ARM64 architecture version of Zen.

### What platform are you seeing the problem on?

Windows

### Additional context

Firefox already ships a Windows ARM64 version.
 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/351_

**Comment by ssalbdivad** [comment]
Given a lot of the next-gen windows laptops are now using ARM-based processors and FireFox already supports it, would love to be able to use Zen on my laptop as well!

**Comment by dam5s** [comment]
Also DRMs will not work on Arm64 when using x64 emulation layer. This means the browser for example cannot read content from HBO or other streaming platform unless there is a native arm64 version.
